WalletWasabi/WabiSabi/Client/CoinJoinCoinSelector.cs

```diff
@@ -63,8 +70,7 @@ public static class CoinJoinCoinSelector
 
 		if (semiPrivateCoins.Length + redCoins.Length == 0)
 		{
-			// Let's not mess up the logs when this function gets called many times.
-			return ImmutableList<TCoin>.Empty;
+			throw new CoinJoinClientException(CoinjoinError.AllCoinsPrivate, "No red coins or semi private coins in the wallet.");
```

Comment:
We could return an empty list here, but to show correct message on the MusicBox I would need to hack in CjManager `LOC451`.